src/routes/api/request.js

```diff
@@ -1,11 +1,18 @@
 import { Router } from 'express';
-import { RequestController } from '../../controllers';
+import { RequestController, } from '../../controllers';
 import validateTripRequest from '../../middlewares/Validations/validateTripRequest';
 
+
 const router = Router();
 
-router.get('/', RequestController.getAllRequests);
+const foo = (io = null) => {
```

Comment:
Hi Ib, I was wondering if exporting this expression allows the router to mount properly?